pandas/core/groupby.py

```diff
@@ -997,16 +997,16 @@ def tail(self, n=5):
         Examples
         --------
 
-        >>> df = DataFrame([[1, 2], [1, 4], [5, 6]],
+        >>> df = DataFrame([['a', 1], ['a', 2], ['b', 1], ['b', 2]],
                             columns=['A', 'B'])
         >>> df.groupby('A', as_index=False).tail(1)
```

Comment:
So the index in the first example is indeed not correct, but for the second example it is OK (there Tom used a wrong groupby key)